FHEM/14_SD_WS_Maverick.pm

```diff
@@ -308,26 +280,44 @@ sub SD_WS_Maverick_SetSensor2Inaktiv($){
   my $name = $hash->{NAME};
   Log3 $hash, 5, "$name SD_WS_Maverick_SetSensor2Inaktiv";
 
-  $hash->{sensor_2_state}="inactiv";
+  $hash->{sensor_2_state}="inactive";
   SD_WS_Maverick_updateReadings($hash);
 }
 
 sub SD_WS_Maverick_updateReadings($){
   my ($hash) = @_;
   my $name = $hash->{NAME};
   Log3 $hash, 5, "$name SD_WS_Maverick_updateReadings";
-  
+
   readingsBeginUpdate($hash);
     readingsBulkUpdate($hash, "temp-food", $hash->{temp_food});
     readingsBulkUpdate($hash, "temp-bbq", $hash->{temp_bbq});
     readingsBulkUpdate($hash, "messageType ", $hash->{messageType});
     readingsBulkUpdate($hash, "checksum", $hash->{checksum});
     readingsBulkUpdate($hash, "Sensor-1-food_state", $hash->{sensor_1_state});
     readingsBulkUpdate($hash, "Sensor-2-bbq_state", $hash->{sensor_2_state});
+    readingsBulkUpdate($hash, "state", $hash->{state});
```

Comment:
Ich würde den State nicht auch noch als internal anlegen.

Reply:
Ich verstehe nicht - hier wird doch kein Internal angelegt, sondern das Reading `state` aktualisiert.

Wenn Du mit dem Kommentar meinst, dass alle in diesem Modul als Readings vorhandenen Werte nicht auch noch zusätzlich in der Form `hash->{foo}` gespeichert werden sollten, dann könnte `updateReadings` komplett entfallen. Ich committe dazu mal einen Vorschlag.

Reply:
Wenn $hash->{state} ein Wert zugewiesen wird, dann führt dass Dazu, dass ein Internal state angelegt wird.
https://wiki.fhem.de/wiki/DevelopmentModuleIntro#Internals 

Für mich hat sich das Kommentar erledigt, da Du es entfernt hast. Wenn Du noch fragen hast, kannst Du dich gerne an mich wenden.